application/common/core.cpp

```diff
@@ -43,19 +44,61 @@ void loop()
 
     // Get the stage as it will receive most events.
     Loom2D::Stage *stage = Loom2D::Stage::smMainStage;
-    
+
+    if(gPendingResizeEvent && stage)
```

Comment:
No, because stage must have valid size from the moment it is instantiated. This is just to fire an event at the next opportunity after the stage has been created. We can't fire it before because then the user code won't be around to receive it.